sonder/utils.py

```diff
@@ -10,3 +10,25 @@ def pgn_to_uci(pgn):
     for move in game.mainline_moves():
         moves.append(move.uci())
     return moves
+
+def import_pgn_to_db(pgn):
+    pgn_in = open(pgn)
+    while True:
```

Comment:
I slightly prefer:

```
     game = chess.pgn.read_game(pgn_in)
     while game:
         into_db(game)
         game = chess.pgn.read_game(pgn_in)
```

Reply:
Won't this miss games since you're read_game()ing twice but only catching one?

Reply:
Shouldn't miss games. it will read the first game from the file before entering the loop. If it's a valid game, then it will insert it into the database and read another game.  If that game is a valid game, then it will insert it and read another one.

Reply:
ohhh the first one starts it off and the second one is to prime it for the next loop, gotcha

Reply:
Correct.